readthedocs/doc_builder/python_environments.py

```diff
@@ -193,12 +193,12 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.5',
+            'sphinx==1.5.3',
```

Comment:
Also +1 on `<1.6` here. The Sphinx devs are very deliberate with versioning these days, I trust that bugfix versions won't have a strong effect here. Additionally, the number of issues we've had from indescriminately selecting versions are outweighted by the number of bugfixes we haven't adopted.